core/commands/refs.go

```diff
@@ -313,6 +313,8 @@ func (rw *RefWriter) WriteEdge(from, to key.Key, linkname string) error {
 
 	var s string
 	switch {
+	case rw.PrintFmt == "<dst>":
```

Comment:
It might be a simple (possibly premature) optimization.

99% of uses of this command will use this format string.